src/templates/search.liquid

```diff
@@ -26,9 +26,7 @@
           {% endif %}
         {% endpaginate %}
       {% else %}
-        <p class="serif-body-l">
-          Looking for something in particular? <u><a href="/#"=>Contact us.</a></u>
-        </p>
+        {% section 'no-search-results' %}
```

Comment:
Hey, I think actually the section should not be included conditionally here, but all the time, and the _copy_ will just be conditionally displayed.

Basically the section can encapsulate the whole search page, which allows the ability to extend it with more settings later, and this conditional can apply to the particular no_search_results setting.

 Otherwise the section settings do not consistently appear in the CMS—it is difficult to access the no-results version of this page from the theme editor